data/json/items/ammo/545x39.json

```diff
@@ -0,0 +1,38 @@
+[
+  {
+    "id": "545", "//" : "Mainlined from Icecoon's Weapons Pack.",
```

Comment:
Since the spaces exist in mainline already, could their removal wait until another PR?